internal/vrf/ips.go

```diff
@@ -0,0 +1,66 @@
+package vrf
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) Ips() *cobra.Command {
+	var (
+		vrfID string
+		ipID  string
+	)
+
+	// createVRFCmd represents the creatVRF command
+	ipsVRFCmd := &cobra.Command{
+		Use:   "ips [-v <vrf-id] [-i <ip-id>]",
+		Short: "Retrieves the list of VRF IP Reservations for the VRF.",
+		Long:  "Retrieves the list of VRF IP Reservations for the VRF.",
+		Example: ` # Retrieves the list of VRF IP Reservations for the VRF..
+	
+	metal vrf ips [-v <vrf-id] 
+
+	# Retrieve a specific IP Reservation for a VRF
+	metal vrf ips [-v <vrf-id] [-i <ip-id>]`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+
+			inc := []string{}
+			exc := []string{}
+
+			var data [][]string
+			var header = []string{"ID", "Type", "ADDRESS", "PUBLIC", "METRO", "Created"}
+
+			if ipID == "" {
+				vrfIpReservationList, _, err := c.Service.FindVrfIpReservations(context.Background(), vrfID).Include(c.Servicer.Includes(inc)).Exclude(c.Servicer.Excludes(exc)).Execute()
+				if err != nil {
+					return fmt.Errorf("could not find VrfIpReservations: %w", err)
+				}
+				ipReservations := vrfIpReservationList.GetIpAddresses()
+				data = make([][]string, len(ipReservations))
+				for i, vrf := range ipReservations {
+					data[i] = []string{vrf.GetId(), string(vrf.GetType()), vrf.GetAddress(), strconv.FormatBool(vrf.GetPublic()), vrf.Metro.GetCode(), vrf.GetCreatedAt().GoString()}
+				}
+			} else {
+				ipReservation, _, err := c.Service.FindVrfIpReservation(context.Background(), vrfID, ipID).Include(c.Servicer.Includes(inc)).Exclude(c.Servicer.Excludes(exc)).Execute()
+				if err != nil {
+					return fmt.Errorf("could not find VrfIpReservation: %w", err)
+				}
+				data = [][]string{{ipReservation.GetId(), string(ipReservation.GetType()), ipReservation.GetAddress(), strconv.FormatBool(ipReservation.GetPublic()), ipReservation.Metro.GetCode(), ipReservation.GetCreatedAt().String()}}
+			}
+
+			return c.Out.Output(nil, header, &data)
+		},
+	}
+
+	ipsVRFCmd.Flags().StringVarP(&vrfID, "vrf-id", "v", "", "Specify the VRF UUID to list its associated IP reservations.")
+	ipsVRFCmd.Flags().StringVarP(&ipID, "ip-id", "i", "", "Specify the IP UUID to retrieve the details of a VRF IP reservation.")
```

Comment:
[The earlier reviews suggested `id` as the name for this parameter](https://github.com/equinix/metal-cli/pull/418#discussion_r1459165178); this is a subcommand for working with IPs, and we generally use the `id` parameter for resources that the subcommand is written for (i.e., the `project` subcommands use `id` for project IDs, `device` subcommands use `id` for devices, etc.).  What is the argument for using `ip-id` instead?

```suggestion
	ipsVRFCmd.Flags().StringVarP(&ipID, "id", "i", "", "Specify the IP UUID to retrieve the details of a VRF IP reservation.")
```